.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,40 @@
+name: Lighthouse
+
+on:
+  pull_request:
+  push:
+    paths-ignore:
+    - 'content/**/*.md'
```

Comment:
Added this line so this workflow won't get triggered if there is a file change in the markdown files. Most users have a fast-laine CI check then :)

Reply:
Perfect!